Framework/include/QualityControl/CustomParameters.h

```diff
@@ -149,6 +149,8 @@ class CustomParameters
 
   std::unordered_map<std::string, std::string>::const_iterator end() const;
 
+  size_t size() const;
```

Comment:
Since `size()` quite non-intuitive in this case (at least to me), could you document that it returns the total count for all the kv pairs for all beam/type combinations?

Reply:
sure, done